src/webviews/apps/plus/graph/GraphWrapper.tsx

```diff
@@ -234,6 +234,12 @@ export function GraphWrapper({
 	const ensuredIds = useRef<Set<string>>(new Set());
 	const ensuredSkippedIds = useRef<Set<string>>(new Set());
 
+	// TODO@ramint Remove/update this logic once markers are individually configurable
+	const disabledScrollMarkers = [GraphScrollMarkerTypes.Tag];
```

Comment:
Need to hook this up with the settings coming through

Reply:
Done in commit: [Use markers in settings](https://github.com/gitkraken/vscode-gitlens/pull/2489/commits/780016d774cce7e278eb5652ce8499d4bd5dbc51)